### DISCUSSION TITLE
Heat on TRV for 100% for minimum of 30 minutes, with cooldown of 30 minutes.

### DISCUSSION THREAD

**Discussion by Eagleman7** [jmcollin78/versatile_thermostat#676] [Q&A] [answered]
Hello!

The way the heating works in my house is that it is "cheaper" to heat for 100% instead of for example 30% on the TRV.  I pay for amount of minutes/hours the heater is on, but not for how "efficient" I heat. Therefor I created on/off switches out of all my Heating TRV's. 

Now I would like to do the following;

1. Heat for a minimum period of 30 or 60 minutes on 100% (to get the most return for the amount of time I am heating my room).
2. Have a cooldown period of at 30 or 60 minutes as well.

I know this will cause the rooms to never have a "stable" temperature because it will overshoot by a lot and then cooldown a lot, however this is the most cost efficient way to heat my rooms.

I was wondering if the cycle duration can help with this?

**Comment by jmcollin78** [comment, marked as answer]
Hello,

No, cycle will not help you.

The VTherm is exactly the contrary of this needs: it manage to get a stable temperature and try to minimize the power consumption. 

You have do your own automation or integration to have this behavior.
